template/settings.html

```diff
@@ -117,3 +117,24 @@ <h3 data-i18n="Carbon_copy"></h3>
       </div>
    </fieldset>
 </form>
+
+<form class="form-horizontal col-sm-6" data-onsubmit="xmpp.chatState.init">
+   <fieldset class="jsxc_fieldsetCarbons jsxc_fieldset">
+      <h3 data-i18n="Chat_state_notifications"></h3>
+      <p data-i18n="setting-explanation-chat-state"></p>
+      <div class="form-group">
+         <div class="col-sm-12">
+            <div class="checkbox">
+               <label>
+                  <input type="checkbox" id="chatState-enable"><span data-i18n="Enable"></span>
+               </label>
+            </div>
+         </div>
+      </div>
+      <div class="form-group">
+         <div class="col-sm-12">
+            <button class="btn btn-primary jsxc_continue" type="submit" data-i18n="Save"></button>
+         </div>
+      </div>
+   </fieldset>
+</form>
```

Comment:
I haven't yet seen, how this will look like, but the current (old) settings window is already taking the whole screen on a 1024x1024 browser window. Are jquery ui tabs an option here?